### PR TITLE
💄(courses) subject detail: template and style

### PR DESCRIPTION
**Pull request by 2018D**
Copy organization detail style. Adapt copied template and style for subject detail. 
Adapt related tests.

## Purpose

To have a basic style for subject detail pages. #220 

## Proposal

Imitate what was done for organization detail. Adjust it to suit subject detail.

### CODE REVIEW

File: src/richie/apps/courses/templates/courses/cms/subject_detail.html [hunk 1]
```diff
@@ -4,49 +4,50 @@
 {% block content %}
 <div class="subject-detail">
```
Comment (sveetch) [inline src/richie/apps/courses/templates/courses/cms/subject_detail.html]: You may add templatetag `{% spaceless%}...{% endspaceless %}` around content just under this div, so HTML is a bit lighter and unnecessary whitespaces won't never make tests fails.

Comment (2018D) [reply src/richie/apps/courses/templates/courses/cms/subject_detail.html]: I ended up not adding the spaceless tag after all.

Comment (sveetch) [reply src/richie/apps/courses/templates/courses/cms/subject_detail.html]: and because ... ?

Comment (2018D) [reply src/richie/apps/courses/templates/courses/cms/subject_detail.html]: I was trying to stay consistent with other templates.
I added them now.
Thanks for the review.

**Comment by sampaccoud** [comment]
@2018D There is a problem on your PR as it is making unwanted changes on courses and organizations. I don't know what happened, maybe a rebase fail? Can you take a look at it?

**Comment by 2018D** [comment]
@sveetch recommanded that i add spaceless tags to the subject template. To be consistent, i also added them to course and organization's templates.

**Pull request merged**